VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionThread.java

```diff
@@ -183,6 +183,8 @@ public void setRead(boolean read) {
 
     public void incrementCommentCount() {
         ++this.commentCount;
+        ++this.unreadCommentCount;
+        setRead(false);
```

Comment:
Instead of calling the `setRead(false)` method, you should just set `read = false` directly, as the method can have different semantics if we introduce a feature for marking a whole thread as unread (e.g. probably _all_ the comments would be set as unread as well in that case).